x-pack/plugins/search_indices/public/components/indices/details_page.tsx

```diff
@@ -156,8 +166,20 @@ export const SearchIndexDetailsPage = () => {
               navigateToIndexListPage={navigateToIndexListPage}
             />
           )}
+          <EuiPageTemplate.Section grow={false}>
+            <EuiFlexGroup>
+              <EuiFlexItem>
+                <ConnectionDetails />
+              </EuiFlexItem>
+              <EuiFlexItem>{/* TODO: API KEY */}</EuiFlexItem>
+            </EuiFlexGroup>
+
+            <EuiSpacer size="l" />
 
-          <div data-test-subj="searchIndexDetailsContent" />
```

Comment:
Is this div and test subject needed for FTRs?

Reply:
Did a quick scan and dont see this being used in our FTRs. My next PR has some checks to see if content is present.